Documentation/docs/learn/faq.md

```diff
@@ -0,0 +1,564 @@
+# Frequenty asked questions
```

Comment:
-> Frequently asked questions (FAQ)

since FAQ is so commonly used?

Reply:
Done in fe3d292.